src/test/java/net/sf/jabref/importer/fileformat/BibTeXMLImporterTest.java

```diff
@@ -53,21 +54,28 @@ public void testGetItemsEmpty() {
 
     @Test
     public void testGetFormatName() {
-        BibTeXMLImporter importer = new BibTeXMLImporter();
         Assert.assertEquals("BibTeXML", importer.getFormatName());
     }
 
     @Test
     public void testGetCLIId() {
-        BibTeXMLImporter importer = new BibTeXMLImporter();
         Assert.assertEquals("bibtexml", importer.getId());
     }
 
     @Test
-    public void testIsRecognizedFormatReject() throws IOException {
-        BibTeXMLImporter importer = new BibTeXMLImporter();
+    public void testsGetExtensions() {
```

Comment:
I think, everything accessible by others should be tested, even though though the tests feel dump. Reason: We are trying to increase code coverage and these tests increase it, too.